renderdoc/driver/d3d12/d3d12_pixelhistory.cpp

```diff
@@ -1128,7 +1135,7 @@ struct D3D12ColorAndStencilCallback : public D3D12PixelHistoryCallback
       D3D12CopyPixelParams depthCopyParams = targetCopyParams;
       depthCopyParams.srcImage = depthImage;
       depthCopyParams.srcImageFormat = GetDepthSRVFormat(depthImage->GetDesc().Format, 0);
```

Comment:
Minor style query : should this be "depthFormat" to match the call to GetDepthCopyFormat()

Reply:
The depth format and image format aren't technically the same, since the DSV could put a different view type on the resource, but GetDepthSRVFormat will turn all compatible aliases into the same type regardless.
I'll change it to depthFormat for consistency.